src/EditModel/event-program/tracker-program/program-stages/actions.js

```diff
@@ -27,9 +27,9 @@ export const changeStep = stepKey => ({
 export const nextStep = () => ({ type: PROGRAM_STAGE_STEP_NEXT });
 export const previousStep = () => ({ type: PROGRAM_STAGE_STEP_PREVIOUS });
 
-export const editProgramStage = stageId => ({
+export const editProgramStage = (stageId, addNewStage = false) => ({
```

Comment:
This is confusing me.

So if I call `editProgramStage(id, true)` then I am adding a new stage? And if I call `editProgramStage(id, false)` I am editing an existing program stage?

Reply:
Exactly. As the function is called `edit`, I thought that the default value should reflect the `edit` state.

You remember correctly, the maintenance app doesn't draw a distinction between editing and adding, I don't like that either. There is a PROGRAM_STAGE_ADD action though, which is used in an epic to create a model with an id and some other things, which then is used to dispatch an edit action to edit that newly created model.

I agree it'd be better to have a proper distinction here... But it seemed to be quite a hassle for a simple wording change.

Reply:
Alright, yeah, might be a cost vs. benefit calculation here. :D